src/testutils/configurable_blockchain_tests.rs

```diff
@@ -234,6 +235,7 @@ where
     builder.add_recipient(address2.script_pubkey(), TRANSFER_AMOUNT / 2);
     let (mut psbt, details) = builder.finish().unwrap();
     assert!(wallet.sign(&mut psbt, Default::default()).unwrap());
+
```

Comment:
Why this new line?

Reply:
fixed. 
My early work was in this file, as it is mentioned in the associated issue.